wholesale_market/remove_date_columns.sql

```diff
@@ -0,0 +1,6 @@
+-- Removes the date columns for wholesale market
+ALTER TABLE wholesale_market DROP FOREIGN KEY wholesale_market_ibfk_3;
+ALTER TABLE wholesale_market DROP COLUMN disposition_date_id;
+ALTER TABLE wholesale_market DROP FOREIGN KEY wholesale_market_ibfk_5;
+ALTER TABLE wholesale_market DROP COLUMN expiration_date_id;
+ALTER TABLE wholesale_market MODIFY COLUMN account_name VARCHAR(100) NOT NULL UNIQUE;
```

Comment:
Same here too. Put this in a separate migration file since it is separate from removing the date columns